config/initializers/aws.rb

```diff
@@ -1,15 +1,16 @@
 CarrierWave.configure do |config|
-  config.fog_provider = 'fog/aws'                     # required
-  config.fog_credentials = {
-    provider:              'AWS',                        # required
-    aws_access_key_id:     ENV['AWS_ACCESS_KEY'],            # required
-    aws_secret_access_key: ENV['AWS_SECRET_KEY'],                        # required
-    region:                'us-west-2',
-    path_style:            true            
-    # host:                  's3.example.com',             # optional, defaults to nil
-    # endpoint:              'https://s3.example.com:8080' # optional, defaults to nil
-  }
-  config.fog_directory  = 'servicealliance'                          # required
-  config.fog_public     = false                                        # optional, defaults to true
-  config.fog_attributes = { 'Cache-Control' => "max-age=#{365.day.to_i}" } # optional, defaults to {}
+  if Rails.env.production?
```

Comment:
> As far as I'm aware, fog won't call AWS unless it's actually trying to upload a file (do correct me if I'm wrong!), and doing this will mean that you won't be able to upload files in development.
> 
> However, for file uploading in dev, you still need credentials to send the file somewhere, and I'm not sure the best way to handle this. We could actually set the storage in the uploaders to :file in dev maybe, possibly by setting the storage method in the application config.
> 
> Am I overthinking this? 📦 
Nope, your right on. This is how fog/carrierwave works by default, using :file. You don't really need to set anything for uploads to work 'locally'.